google-cloud-storage/src/main/java/com/google/cloud/storage/transfermanager/DirectDownloadCallable.java

```diff
@@ -50,16 +49,26 @@ final class DirectDownloadCallable implements Callable<DownloadResult> {
   @Override
   public DownloadResult call() {
     Path path = TransferManagerUtils.createDestPath(parallelDownloadConfig, originalBlob);
-    try (ReadChannel rc = storage.reader(originalBlob.getBlobId(), opts)) {
+    long bytesCopied = 0L;
```

Comment:
It would be better to initialize this to `-1` or something less than zero instead of `0`. There can be objects that are `0` bytes, so that would actually be a success, as long as the file is created on disk (something else to double check).